code/game/jobs/job/civilian.dm

```diff
@@ -221,7 +223,7 @@
 	selection_color = "#dddddd"
 	access = list(access_journalist, access_maint_tunnels)
 	minimal_access = list(access_journalist, access_maint_tunnels)
-	alt_titles = list("Freelance Journalist")
+	alt_titles = list("Freelance Journalist", "Newscaster Maintainer")

```

Comment:
What is the purpose of the newscaster maintainer ?

We have corporate reporters, that are employed by NT to report on the station events positively
We have the freelance journalist, from a external news organization with more independence than the corporate reporter.

Given that we have one role that is employed by NT and another one that is employed by a external news org.
What is the difference to the newscaster maintainer, and what is their status in regards to the journalist directive ?

Reply:
Newscasters aren't _always_ news - I've seen people create newscaster feeds with riddles or similar. I'd like that to migrate to the journalist profession instead of librarians doing it.